include/deal.II/dofs/dof_accessor.h

```diff
@@ -1588,6 +1588,9 @@ class DoFCellAccessor :  public DoFAccessor<DH::dimension,DH, level_dof_access>
    * @{
    */
 
+  // Necessary for Visual Studio for not to replace the get_fe version with one parameter with the following parameter-less version.
+  using DoFAccessor<DH::dimension, DH, level_dof_access>::get_fe;
+
```

Comment:
This actually has not happen until I made those ugly changes that made the code work on VS. Then this issue came up. I guess before this, the DoFHandlers got all mixed up and it did not come to this issue.